app/src/utils/data/ipfs.ts

```diff
@@ -6,7 +6,7 @@ import { BigNumber } from 'src/utils/ethers';
 import { LocalForageData } from 'src/types';
 import { Ref } from 'vue';
 
-const retrievalEndpoint = 'https://ipfs.fleek.co/ipfs';
+const retrievalEndpoint = 'https://scopelift.b-cdn.net/ipfs/<cid>/';
```

Comment:
The `/<cid>/` part was meant to be a placeholder for the content's CID.

Reply:
Lol, I just commented on this - I'll delete it now! 🚀

Reply:
Clearly I reviewed too quickly, whoops. Thanks all